### ISSUE TITLE
horizontal/vertical-viewport-segments - No data for the viewport-segments media queries

### ISSUE THREAD

**Issue by lukewarlow** [mdn/browser-compat-data#22227]
### What type of issue is this?

Missing compatibility data

### What information was incorrect, unhelpful, or incomplete?

There's no support data for the two viewport-segments media queries. See https://drafts.csswg.org/mediaqueries-5/#mf-horizontal-viewport-segments for spec.

### What did you expect to see?

Chromium has support for these media queries behind a flag and I believe both Edge and Samsung Internet have them shipped to stable.

### Did you test this? If so, how?

Yes using https://mq5.goose.icu/#viewport/page%20characteristics

**Comment by foolip** [comment]
The media query features are now in BCD, but the related `env()` stuff here isn't:
https://drafts.csswg.org/css-env-1/#viewport-segments

**Comment by foolip** [comment]
No, wait, it was added in https://github.com/mdn/browser-compat-data/pull/27387.

@lukewarlow is this all done then?

**Comment by lukewarlow** [comment]
Yeah looks like it. Just need content pages now I believe.
